Adyen/UI/Form/Items/Basic Items/FormAttributedLabelItem.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Simple form item that represent a single attributed UILabel element.
+@_spi(AdyenInternal)
+public class FormAttributedLabelItem: FormItem {
+
+    public var subitems: [FormItem] = []
+
+    public init(tAndCText: String, link: String, style: TextStyle, identifier: String? = nil) {
+        self.identifier = identifier
+        self.style = style
+        self.tAndCText = tAndCText
+        self.link = link
+    }
+
+    public var identifier: String?
+
+    /// The style of the label.
+    public var style: TextStyle
+
+    /// The text of the label.
+    public var tAndCText: String
```

Comment:
please avoid abbreviations :-), its hard to read

Reply:
Okay.  I will rename this.